docs/asciidoc/modules/ROOT/pages/operational/init-script.adoc

```diff
@@ -7,7 +7,7 @@
 Apoc optionally allows you to run a cypher command after database initialization is finished.
 This can e.g. be used to ensure indexes/constraints are created up front.
 
-To use this feature use a config option:
+To use this feature, add the following line to Neo4j configuration file:
```

Comment:
it doesn't have to be necessarily in the config file. We can use either `apoc.conf`, env variables or system properties

Reply:
words are hard

Reply:
I have added the link to the configuration page

Reply:
Obviously I don't know how to link properly, if you could help me with that would be great!